vendor/github.com/livepeer/lpms/stream/basic_rtmp_videostream.go

```diff
@@ -62,8 +63,6 @@ func (s *BasicRTMPVideoStream) GetStreamFormat() VideoFormat {
 
 //ReadRTMPFromStream reads the content from the RTMP stream out into the dst.
 func (s *BasicRTMPVideoStream) ReadRTMPFromStream(ctx context.Context, dst av.MuxCloser) (eof chan struct{}, err error) {
-	defer dst.Close()
```

Comment:
If we don't defer the close here is the user just responsible for closing the MuxCloser? Is there anywhere that we use `ReadRTMPFromStream` that we now need to make sure that MuxCloser is closed?

Reply:
I think it makes sense for the users to be responsible for closing the mux (if they are the ones passing it in)